src/resolvelib/resolvers.py

```diff
@@ -9,8 +9,12 @@
     Collection,
     Generic,
     Iterable,
+    List,
```

Comment:
You can use `list` and `tuple` directly since `annotations` has been imported.

Reply:
Fixed